web/src/main/java/com/epam/esm/controller/OrderController.java

```diff
@@ -26,32 +28,41 @@ public OrderController(OrderService orderService, OrderModelAssembler orderModel
     @GetMapping(value = "/{id}")
     @PreAuthorize("hasAnyRole('USER','ADMIN')")
     public OrderModel getOrder(@PathVariable("id") Long id) {
-        return orderModelAssembler.toModel(orderService.findById(id)
-                .orElseThrow(() -> new NotFoundException("Order not found with id: " + id)));
+        return orderModelAssembler.toModel(orderService.findById(id));
```

Comment:
According to this implementation, users also can see all other users' orders. It seems to me wrong